titles/plugin-rhdh/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Configuring plugins in {product}</title>
+<productname>{product}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>
+</subtitle>
+<abstract>
+    <para>Red Hat Developer Hub is a developer platform for building developer portals. This document explains how you can configure plugins in {product}.</para>
```

Comment:
CCS style guide: _Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​"._ Consider something like the following: 

```suggestion
    <para>Red Hat Developer Hub is a developer platform for building developer portals. You can create new plugins to customize your {product-short} experience.</para>
```

Reply:
Changed the sentence that aligns with the book content.